packages/astro/test/no-head-el.test.js

```diff
@@ -25,4 +25,14 @@ NoHeadEl('Places style and scripts before the first non-head element', async ({
   assert.equal($('script[src="/_snowpack/hmr-client.js"]').length, 1, 'Only the hmr client for the page');
 });
 
+NoHeadEl('Places style and scripts before the first non-head element even when there are no elements on the page', async ({ runtime }) => {
```

Comment:
How does this test work? Where does it specify the location of style/scripts?

Reply:
The hmr script is added automatically, the test doesn't have to specify where it goes.

Reply:
Yeah the description was copy + pasted, I updated it to clarify!